tutorials/roofit/rf902_numgenconfig.py

```diff
@@ -18,7 +18,7 @@
 # Example pdf for use below
 x = ROOT.RooRealVar("x", "x", 0, 10)
 model = ROOT.RooChebychev(
-    "model", "model", x, ROOT.RooArgList(ROOT.RooFit.RooConst(0), ROOT.RooFit.RooConst(0.5), ROOT.RooFit.RooConst(-0.1))
+    "model", "model", x, [0, 0.5, -0.1]
```

Comment:
```suggestion
    "model", "model", x, [0.0, 0.5, -0.1]
```
Same problem as in rf313_paramranges.py.

By the way did you remember to run `black --line-length=120`? I'm surprised that the code formatting didn't put the whole `RooChebychev` construction in one line.